tools/wptrunner/wptrunner/tests/test_testloader.py

```diff
@@ -18,6 +23,50 @@
 """
 
 
+def test_loader_h2_tests():
+    manifest_json = {
+        "items": {
+            "testharness": {
+                "a": {
+                    "foo.html": [
+                        "abcdef123456",
+                        # TODO: This isn't what MANIFEST.json looks like. It
```

Comment:
@jgraham I'm having some trouble figuring out what is going wrong here. Any thoughts?

For completeness, the stack trace to the URLManifestItem constructor is:

```
wptrunner/testloader.py:247: in __init__
    self._load_tests()
wptrunner/testloader.py:310: in _load_tests
    for test_path, test_type, test in self.iter_tests():
wptrunner/testloader.py:291: in iter_tests
    manifest_items.extend(manifest_iter)
wptrunner/testloader.py:196: in iterfilter
    for item in iter:
../manifest/manifest.py:147: in itertypes
    tests = self._data[item_type][path]
../manifest/typedata.py:106: in __getitem__
    manifest_item = self._type_cls.from_json(self._manifest, path, test)
../manifest/item.py:173: in from_json
    **extras)
```

Reply:
You want `None` for the JSON `null` value, otherwise it'll handle it as a string

Reply:
You actually want `[None, {}}` here since it's `url, extras`